Detections/MultipleDataSources/powershell_MangoSandstorm.yaml

```diff
@@ -29,39 +29,50 @@ query: |
   | where EventID == 4688
   | where Process has_any ("powershell.exe","powershell_ise.exe","pwsh.exe") and CommandLine has_cs "-exec bypass -w 1 -enc"
   | where CommandLine  contains_cs "UwB0AGEAcgB0AC0ASgBvAGIAIAAtAFMAYwByAGkAcAB0AEIAbABvAGMAawAgAHsAKABzAGEAcABzACAAKAAiAHAA"
-  | extend timestamp = TimeGenerated, AccountCustomEntity = Account, HostCustomEntity = Computer, ProcessCustomEntity = Process
+  | extend timestamp = TimeGenerated
   ),
   (DeviceProcessEvents
   | where FileName =~ "powershell.exe" and ProcessCommandLine has_cs "-exec bypass -w 1 -enc"  
   | where ProcessCommandLine contains_cs "UwB0AGEAcgB0AC0ASgBvAGIAIAAtAFMAYwByAGkAcAB0AEIAbABvAGMAawAgAHsAKABzAGEAcABzACAAKAAiAHAA" 
-  | extend timestamp = TimeGenerated, AccountCustomEntity = AccountName, HostCustomEntity = DeviceName, ProcessCustomEntity = InitiatingProcessFileName
```

Comment:
You will have to map DeviceName to DvcHostName otherwise your HostName and HostNameDomain values won't populate for this since you use the field DvcHostName.

You will also need to map the proper field to get the ProcessID, InitiatingProcessFileName is the name of the process not the id (number) associated with it.  The column is probably called InitiatingProcessId or something like that.